core/commands/storage/upload/upload/init.go

```diff
@@ -26,6 +26,11 @@ import (
 	"github.com/cenkalti/backoff/v3"
 	cidlib "github.com/ipfs/go-cid"
 	ic "github.com/libp2p/go-libp2p-core/crypto"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+const (
+	challengerPid = "16Uiu2HAm3Mw7YsZS3f5KH8VS4fnmdKxgQ1NNPugX7DpM5TQZP9uw"
```

Comment:
this is very fragile, guard will have multiple repair nodes and pids soon (and they can be replaced at any time), so it will be less effective this way.

Reply:
i see, it's just a temporary workaround imo.